label_sync/Makefile

```diff
@@ -0,0 +1,59 @@
+# Copyright 2017 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+all: build test
+
+
+LABEL_SYNC_VERSION       ?= 0.100
+
+# These are the usual GKE variables.
+PROJECT       ?= k8s-prow
+BUILD_PROJECT ?= k8s-prow-builds
+ZONE          ?= us-central1-f
+CLUSTER       ?= prow
+
+# Build and push specific variables.
+REGISTRY ?= gcr.io
+PUSH     ?= gcloud docker -- push
+
+DOCKER_LABELS=--label io.k8s.prow.git-describe="$(shell git describe --tags --always --dirty)"
```

Comment:
I'd prefer to see this without the `--label` as part of the variable and later do

```shell
cmd --labels "$DOCKER_LABELS"
```

Reply:
I was following example in `prow/Makefile`. Please note that I cannot test this, I'm testing locally. I don't know the standars/workflow for creating new commands in `test-infra` and I found no tutorial about this.